### ISSUE TITLE
Rendering artifacts

### ISSUE THREAD

**Issue by tolmasky** [subethaedit/SubEthaEdit#88]
There seem to be rendering artifacts that show up:

1. Sometimes a highlight isn't completely cleared leaving a thing blue outline where the highlight used to be.
2. Sometimes the cursor remains after moving it somewhere else.

**Comment by monkeydom** [comment]
yeah - I noticed this sometimes too. Can you also add the information from the SEE bug report form about your machine, os and setup for this? I do think it is happening more for certain machines.

**Comment by monkeydom** [comment]
Does this still happen or is this gone now?

**Comment by tolmasky** [comment]
I believe this still does happen for me, will report back in a few days of using the release build.
